kotlin/internal/defs.bzl

```diff
@@ -37,6 +37,7 @@ KtJvmInfo = provider(
         "outputs": "output jars produced by this rule. [intelij-aspect]",
         "language_version": "version of kotlin used. [intellij-aspect]",
         "transitive_compile_time_jars": "Returns the transitive set of Jars required to build the target. [intellij-aspect]",
+        "transitive_runtime_jars": "Returns the transitive set of Jars required to run the target. [intellij-aspect]",
```

Comment:
I believe you can already get this from the `JavaInfo` provider that will exist alongside the `KtJvmInfo` provider. Is that not the case?